net/libreswan/patches/040-disable_man.patch

```diff
@@ -1,10 +1,10 @@
 --- a/mk/targets.mk
 +++ b/mk/targets.mk
-@@ -64,7 +64,7 @@ endef
+@@ -65,7 +65,7 @@ endef
  #
  # For each define: TARGET clean-TARGET install-TARGET
  
--TARGETS = base manpages
+-TARGETS = base manpages html
 +TARGETS = base
```

Comment:
I think, I am not sure if we really need this patch, though. 
While looking at libreswan's Makefile, I see that there is a possibility to have minimal installation, which basically you are doing, but in a way, when you need to rebase this patch every time.

Can you please check this: https://github.com/libreswan/libreswan/blob/bee7f36d480dd3912a5135a933b72025c819bcbf/Makefile#L50

and try to call make install-base? :)

Reply:
sure but the build sistem is not the friendliest on and trying something brief didn't lead to success, in short I don't know how to pass make base to the build step, I could switch to replace with sed in the Makefile and not have a patch file